app/binance/user.js

```diff
@@ -67,15 +75,19 @@ const setupUserWebsocket = async logger => {
         if (_.isEmpty(lastOrder) === false) {
           // Skip if the orderId is not match with the existing orderId
           // or Skip if the transaction time is older than the existing order transaction time
-          // This is helpful when we received a delayed event for any reason
+          // Skip if if the creation time is older than the existing order creation time
```

Comment:
I think this comment is not valid because the change was reverted.

Reply:
Ah yes, I reverted but forgot that line.
I revert that as well.